### PR TITLE
DEVDOCS-4398: [net new] Tax Settings API

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4398](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4398)

### CODE REVIEW

File: reference/tax_settings.v3.yml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      summary: Get Tax Settings
+      description: Retrieves global-level tax settings.
+      operationId: get-tax-settings
+      parameters:
+        - $ref: '#/components/parameters/storehash'
+      responses:
+        '200':
+          description: OK
+          content:
+            '*/*':
```
Comment (bc-traciporter) [inline reference/tax_settings.v3.yml:24]:
```suggestion
            '*/*': Do we need this?
```

Comment (bc-andreadao) [reply reference/tax_settings.v3.yml:24]: i think it was a placeholder for content type, so i changed it to "application/json"

File: reference/tax_settings.v3.yml [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        price_display_settings:
+          type: object
+          description: Settings that describe how prices are displayed at the global level.
+          properties:
+            show_inclusive_in_control_panel:
+              type: boolean
+              description: Whether to show prices as tax inclusive or tax exclusive in the BigCommerce control panel.
+            invoice_price_display_strategy:
+              type: string
+              description: 'Whether to show prices as tax inclusive or tax exclusive across all invoices, or use the shopper''s tax zone for price display on invoices.'
```
[the remark this answers is not in the document]
Comment (theromulans) [reply reference/tax_settings.v3.yml:103]: ^ Careful here, I think the single quote might affect the parsing?

**Review by theromulans — DISMISSED** [review]
Thanks @bc-andreadao

**Comment by bc-andreadao** [comment]
Accidentally undid some of Traci's changes somehow in Stoplight, so redid them

**Comment by bc-andreadao** [comment]
Accidentally undid some of Traci's changes somehow in Stoplight, so redid them

File: reference/tax_settings.v3.yml [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Tax_Settings_Req'
+        required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            '*/*':
```
Comment (markcmurphy) [inline reference/tax_settings.v3.yml:50]:
```suggestion
            'application/json:':
```

**Review by markcmurphy — APPROVED** [review]

**Pull request merged**